packages/lambdas/src/sqs-to-fhir.ts

```diff
@@ -292,3 +301,18 @@ function processFHIRResponse(
     });
   }
 }
+
+async function changeMessageVisibility(record: SQSRecord, timeout: number) {
+  const command = new ChangeMessageVisibilityCommand({
+    QueueUrl: record.eventSourceARN,
+    ReceiptHandle: record.receiptHandle,
+    VisibilityTimeout: timeout,
+  });
+
+  try {
+    await sqsClient.send(command);
+  } catch (error) {
+    console.error("Error changing message visibility:", error);
+    // You might want to handle this error or rethrow it depending on your requirements
+  }
+}
```

Comment:
If we keep this approach/solution, let's move this to a shared file? I think there's one there for SQS already.